addons/binding/org.openhab.binding.homie/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Homie Binding
+Bundle-SymbolicName: org.openhab.binding.homie;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,lib/commons-lang3-3.5.jar
```

Comment:
1) I opened an issue for that, but are you using anything specific in this case? https://github.com/eclipse/smarthome/issues/3522
2) I did not check the exact impact but the idea of the transport bundle is that not all bindings have to duplicate the transport handling and drag along there own jar's. if you require changes in the transport bundle you can of course discuss/suggest them and create a PR